docs/api-reference/next/future/image.md

```diff
@@ -101,6 +101,18 @@ const MyImage = (props) => {
 }
 ```
 
+### fill
+
+A boolean attribute. If present, sets the image to fill its containing element rather than defining its own size. When using `fill` mode, do not provide `height` and `width`.
+
+When using the `fill` attribute, the parent element _must_ have the CSS `position: "relative"` or the image will not render correctly. Also note that the image itself is automatically given `position: "absolute"` and this value should not be changed.
```

Comment:
I thought we wanted to avoid making the user manually set `position: "relative"`.

I don't recall seeing this in the "Proposal: Fill behavior for next/image" document.

 cc @cramforce @kara

Reply:
So, there's one specific case that's the problem, where the user uses the `fill` mode on the image, but does not include height or width styling on the container element. In this case, the container element takes the intrinsic size of the image, even though `object-fit: "contain"` is set. This will cause layout shift.

This doesn't happen if the `img` element has `position: "absolute"` and the container element has `position: "relative"`. I haven't found any other techniques for keeping the unsized container from taking the size of the image.